python/scripts/ingest_eeg_bids_datasets.py

```diff
@@ -127,13 +127,13 @@ def main():
         if os.path.isdir(
             os.path.join(assembly_bids_path, 'sub-' + str(candid))
         ):
-            subjectid = str(candid)
+            subjectid = 'sub-{}'.format(candid)
```

Comment:
```suggestion
            subjectid = f'sub-{candid}'
```

(@jeffersoncasimir to fix the test errors)

Reply:
thanks